components/server/ee/src/workspace/gitpod-server-impl.ts

```diff
@@ -2146,35 +2146,64 @@ export class GitpodServerEEImpl extends GitpodServerImpl {
         }
     }
 
-    async getUsageLimitForTeam(ctx: TraceContext, teamId: string): Promise<number | undefined> {
-        const user = this.checkAndBlockUser("getUsageLimitForTeam");
-        const team = await this.guardTeamOperation(teamId, "get");
-        await this.ensureStripeApiIsAllowed({ team });
+    async getUsageLimitFor(ctx: TraceContext, attributionId: string): Promise<number | undefined> {
+        const attrId = AttributionId.parse(attributionId);
+        if (attrId == undefined) {
```

Comment:
nit: `==` might do the trick here, but it instantly reads like a (potential) bug. We tend to use `===` everywhere.